### ISSUE TITLE
running the tests is a bit tedious 

### ISSUE THREAD

**Issue by kareemmahlees** [piccolo-orm/piccolo#737]
Hey, I am interested in contributing to piccolo.
### The issue
I created a Postgres database in a docker container and adjusted the settings in `tests\postgres_conf.py` as stated [here](https://piccolo-orm.readthedocs.io/en/latest/piccolo/contributing/index.html#:~:text=Setup%20Postgres%2C%20and%20make%20sure%20a%20database%20called%20piccolo%20exists%20(see%20tests/postgres_conf.py).).
but when i run :
````console
pytest -s -v
````
i get this error :
````console
ImportError while loading conftest 'D:\Programming\Code_projects\piccolo\tests\conftest.py'.
tests\conftest.py:5: in <module>
    ENGINE = engine_finder()
piccolo\engine\finder.py:19: in engine_finder
    return Finder().get_engine(module_name=module_name)
piccolo\conf\apps.py:399: in get_engine
    piccolo_conf = self.get_piccolo_conf_module(module_name=module_name)
piccolo\conf\apps.py:362: in get_piccolo_conf_module
    module = t.cast(PiccoloConfModule, import_module(module_name))
piccolo_conf.py:14: in <module>
    DB = PostgresEngine(config={})
. . . . . . . . 
E   asyncpg.exceptions.InvalidPasswordError: password authentication failed for user "kareem"
````
I can see that it looks for a module called piccolo_conf but I can't figure out where that module resides as the stack trace doesn't show the path of that file  ( even after digging into the code ).
I tried setting the`module_name` argument in `conftest.py` but I am not sure how the paths are treated or to which package they are taken relative to 
### TL;DR 
I would like clarification on where I should put the `*_conf.py` file so it can be detected by the importer and utilized, or, how to define the whole path and relative to what package.

**Comment by sinisaos** [comment]
@kareemmahlees Thanks for interest in Piccolo. From the stack trace I see you are running Windows OS. I'm no Windows expert, but if you look inside `piccolo/scripts/test-postgres.sh` you'll see that we need to export the environment variables from `postgres_conf.py` first. I don't know how to do it in Windows (maybe [this](https://linuxhint.com/windows-equivalent-export-command)). After that you can try to run the pytest command from `piccolo/scripts/test-postgres.sh`. If that doesn't work, maybe you can try [WSL for Windows](https://linuxhint.com/run-sh-file-windows/) and then use `./scripts/test-postgres.sh` to export enviroment variables and run tests in one command (like in Linux system). Hope this helps.

**Comment by kareemmahlees** [comment]
@sinisaos Thank you this worked.

**Comment by sinisaos** [comment]
@kareemmahlees Great. Thank you for your interest in Piccolo
